extrinsic_matrix:
- [ -0.48196655, -0.87617352, -0.00531022, -0.01431310 ]
- [  0.03429784, -0.01280996, -0.99932956,  0.03173908 ]
- [  0.87551808, -0.48182555,  0.03622483, -0.03380000 ]
- [  0.0,         0.0,         0.0,         1.0        ]

